db/schema.rb

```diff
@@ -322,6 +370,8 @@
   add_foreign_key "pages", "liquid_layouts", column: "secondary_liquid_layout_id"
   add_foreign_key "plugins_actions", "forms"
   add_foreign_key "plugins_actions", "pages"
+  add_foreign_key "plugins_fundraisers", "forms"
+  add_foreign_key "plugins_fundraisers", "pages"
   add_foreign_key "plugins_thermometers", "pages"
   add_foreign_key "share_emails", "pages"
   add_foreign_key "share_facebooks", "images"
```

Comment:
A little concerned with this change to schema. Could you remove it from this PR?